go-controller/pkg/ovn/hybrid.go

```diff
@@ -259,65 +163,17 @@ func (m *MasterController) handleOverlayPort(node *kapi.Node, annotator kube.Ann
 	return nil
 }
 
-func (m *MasterController) deleteOverlayPort(node *kapi.Node) {
+func (oc *Controller) deleteOverlayPort(node *kapi.Node) {
```

Comment:
@JacobTanenbaum let's make this `deleteHybridOverlayPort()`.

Reply:
done